lib/lib/deck.layer.lib.ts

```diff
@@ -0,0 +1,786 @@
+import type {
+  ArcLayerProps,
+  GeoJsonLayerProps,
+  BitmapLayerProps,
+  ColumnLayerProps,
+  LineLayerProps,
+  PolygonLayerProps,
+  ScatterplotLayerProps,
+  PathLayerProps,
+  IconLayerProps,
+  ArcLayer,
+  ColumnLayer,
+  GeoJsonLayer,
+  BitmapLayer,
+  LineLayer,
+  PolygonLayer,
+  ScatterplotLayer,
+  PathLayer,
+  IconLayer,
+} from "@deck.gl/layers";
+
+import { type ComponentPropsOptions, type PropType } from "vue";
+import {
+  COORDINATE_SYSTEM,
+  LayerExtension,
+  type PickingInfo,
+  type Color,
+  type Unit,
+  type Position,
+  type LayerProps,
+  type LayerData,
+  type Accessor,
+  type Material,
+} from "@deck.gl/core";
+import type { MjolnirEvent } from "mjolnir.js";
```

Comment:
It shouldn't a dependency? Or is it possible to use another type?

Reply:
**mjolnir.js** is an indirect dependency, it is installed as part of the dependency tree of **Deck.gl**.
[see this](https://github.com/visgl/deck.gl/blob/master/docs/upgrade-guide.md#user-input-handling)